src/app/(admin)/admin/knowledge/components/KnowledgeEditor.tsx

```
@@ -122,6 +205,82 @@ export function KnowledgeEditor({ entry, onSave, onCancel }: KnowledgeEditorProp
+            value={value ? new Date(value).toISOString().split('T')[0] : ''}
+            onChange={(e) => handleMetadataChange(key, e.target.value ? new Date(e.target.value).toISOString() : '')}
+            className="flex-1 px-3 py-2 border border-gray-300 rounded-lg focus:ring-2 focus:ring-blue-500 focus:border-blue-500"
+          />
+        );
+      case 'tags':
+        const tagsArray = Array.isArray(value) ? value : (typeof value === 'string' ? value.split(',').map(t => t.trim()).filter(Boolean) : []);
+        return (
+          <div className="flex-1">
+            <input
+              type="text"
+              value={tagsArray.join(', ')}
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)